examples/complete-ecs/main.tf

```diff
@@ -0,0 +1,58 @@
+provider "aws" {
+  region  = "eu-west-1"
+  version = "v1.18.0"
```

Comment:
No need to specify `version` here, because we need to be able to run examples using latest automatically.